src/app/pages/FrontPage/FrontPage.jsx

```diff
@@ -61,14 +62,6 @@ const StyledRadioScheduleContainer = styled(RadioScheduleContainer)`
   ${negativeMargin}
 `;
 
-const MPUContainer = styled(AdContainer)`
-  ${negativeMargin}
```

Comment:
I don't think the `negativeMargin` variable is needed anymore: https://github.com/bbc/simorgh/pull/8019/files#diff-f7b436d13f89521c964d6f76a426bb59R47, I think you could just move its contents into here: https://github.com/bbc/simorgh/pull/8019/files#diff-f7b436d13f89521c964d6f76a426bb59R62

Arguably if this CSS is shared between `StyledRadioScheduleContainer` and `MPUContainer` it should be extracted into a shared const somewhere and used in both places, did you consider this?

Reply:
Good idea, I'll do this now.